builds/releasenotes.md

```diff
@@ -7,11 +7,50 @@ Known Major Issues In Current Release
   - Images in the scene are not using thumbnails (EVC-513)
 
 Version 0.4.0 (WIP)
-  - Improvements
-    - ?
-  - Other Fixes and Changes
-    - ?
-
+  - Conversion Improvements
+    - Fixed issue with resetting some conversion types
+    - Increased support for additional types of FBX files
+    - Added drop down to set all spaces in a conversion rather than doing it individually
+    - Convert CMD takes a global offset now (`-globaloffset X Y Z`)
+    - Added Earth Centered, Earth Fixed support to conversion
+    - Fixed some issues with estimation of resolution for conversion
+    - Added support for additional SRID codes from customers and the community
+  - Improvements to SLPK
+    - Loading is ordered to focus on things closer to the camera
+    - Fixed some crahes that occurred when loading lots of SLPK files at the same time from fast disks
+    - Added ability to reset I3S models positions from the context menu
+  - Various rendering improvements
+    - The scene is scaled properly to avoid aliasing affects
+    - Improved performance when moving the mouse over the scene window with lots of polygon objects
+    - Added basic lighting to Polygon models (including existing Live Feed and SLPK files)
+  - macOS Fixes
+    - Improved consistency of visualisation to match OpenGL and D3D
```

Comment:
This doesn't mention Metal - I believe we still ship OpenGL for macOS?

Reply:
True- We've never shipped Metal before either.